spec/integration/outputs/parent_spec.rb

```diff
@@ -1,7 +1,8 @@
 require_relative "../../../spec/es_spec_helper"
 require "logstash/outputs/elasticsearch"
 
-context "when using elasticsearch 5.x and before", :integration => true, :version_less_than_equal_to_5x => true do
+if ESHelper.es_version_satisfies?("<= 5.x")
+context "when using elasticsearch 5.x and before", :integration => true do
```

Comment:
I just put an explanation on top: I didn't indent these lines on purpose at this WIP stage of the PR, so it's much easier to see the changes without whitespace noise.